OpenAI/Packages/com.openai.unity/Runtime/OpenAIClient.cs

```diff
@@ -64,7 +64,7 @@ protected override void SetupDefaultRequestHeaders()
 #if !UNITY_WEBGL
                 { "User-Agent", "com.openai.unity" },
 #endif
-                { "OpenAI-Beta", "assistants=v1"}
+                { "OpenAI-Beta", "assistants=v2"}
```

Comment:
these changes should be broken out into a separate PR from the projectId changes